sourcegraph/sourcegraph
# Refactor code-intel extensions bundling for VSCE and browser extension

Refactors to use already generated code intel extension bundles from [sourcegraph-extensions-bundles](https://github.com/sourcegraph/sourcegraph-extensions-bundles) instead of generating them from  from [sourcegraph/code-intel-extensions](https://github.com/sourcegraph/code-intel-extensions). 
More context in [this thread](https://sourcegraph.slack.com/archives/C01LZKLRF0C/p1659507269622679).

## Test plan
- VSCE
  - ensure code-intel works
- Browser extension
  - ensure code-intel works for Firefox (logic didn't change for Chrome and Safari)
  - ensure `create-source-zip` script includes bundled extensions in the output
 



## App preview:

- [Web](https://sg-web-taras-yemets-refactor-code-intel.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-fpnexrrwha.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
